apistos/src/app.rs

```diff
@@ -60,13 +60,20 @@ pub struct App<T> {
 #[derive(Default)]
 pub struct BuildConfig {
   ui_plugin_configs: Vec<Box<dyn UIPluginConfig>>,
+  spec_prefix: String,
 }
 
 impl BuildConfig {
   pub fn with<T: UIPluginConfig + 'static>(mut self, plugin: T) -> Self {
     self.ui_plugin_configs.push(Box::new(plugin));
     self
   }
+
+  /// Set the prefix for the openapi spec path
+  pub fn with_spec_prefix<T: Into<String>>(mut self, prefix: T) -> Self {
+    self.spec_prefix = prefix.into();
+    self
+  }
```

Comment:
It might be more versatile to completely override the spec path used by the ui plugin there instead of adding a prefix. What do you think ?

Reply:
Yes you're right !!

Reply:
I'll try to find a way in to avoid breaking current behavior

Reply:
Should be good, without breaking change!